docs/bindings/nodejs/releasing.md

```diff
@@ -0,0 +1,82 @@
+# Releasing
+
+Releasing a new version of `node-valhalla` is mostly automated using CircleCI.
+
+// TODO: HOW TO VERSION???
+
+These steps all happen on `master`. After the release is out, create a branch using the MAJOR.MINOR version of the release to document code changes made for that version.
+
+### Steps to release
+
+1. Update the `package.json` to the correct version.
+
+   Confirm the desired Valhalla branch and commit to `master`. Update the `CHANGELOG.md`.
+
+1. Check that CircleCI for the latest commit on `master`.
+
+1. Publishing binaries
+
+   If circle builds are passing then it's time to publish binaries by committing with a message containing `[publish binary]`. Use an empty commit for this.
```

Comment:
so in OSRM, we used a regex for `tags` I believe. When a tag was something like `vMAJOR.MINOR.PATCH`, Circle CI would build, publish, binaries, so you could consolidate this step with the `1. Tag` step we're mentioning later?

Reply:
also found that @TheMarex already added some release info here - https://github.com/valhalla/valhalla/blob/master/docs/releasing.md

I will try to consolidate any info in here that isn't already in there!